### PR TITLE
Compiled files be gone + meta tag update

### PR DESCRIPTION
**Pull request by ascott1**
## Changes

- Removes the compiled `.js` and `.css` files (hopefully for good)
- Switches the IE meta tag to a single tag
- Updates the README to include grunt instructions and a quick start

## Why?

- The meta tag and removing compiled source are holdovers from closed #690
- README updates were requested by @cmc333333 as compiling front end assets is required to run the app

## Review

- @kimberlymunoz

### CODE REVIEW

**Comment by ascott1** [comment]
Just added some README updates as well.

**Comment by KimberlyMunoz** [comment]
Can you also remove the `style.css.min` file. It's been causing so many conflicts.

File: README.md [hunk 1]
```diff
@@ -159,23 +136,68 @@ In the root of the repository, copy ```example-config.json``` to ```config.json`
...
-###### For functional tests
+##### For functional tests
 - They also require having the environment serving data from ```dummy_api/```. To start the dummy API, from the root of your repo, run ```./dummy_api/start.sh 0.0.0.0:8282```.
 - The tests run using [nose](http://nose.readthedocs.org/en/latest/). If you wish to run the tests outside of the Grunt environment, you may by running ```nosetests regulations/uitests/*.py``` from the root of the repo.
 
-###### For unit tests
+##### For unit tests
 - To run unit tests individually: ```regulations/static/regulations/js/unittests/sauce_unit_tests.sh http://url.of/test/site``` from the root of the repo.
 - Unit tests do not require running the dummy API.
 - You may also run the unit tests locally with no additional configuration by loading the following URL in a web browser: ```http://your.site/static/regulations/js/unittests/runner.html```.
```
Comment (KimberlyMunoz) [inline README.md]: The html runner has been replaced with a node runner on `grunt test`

**Comment by KimberlyMunoz** [comment]
Everything looks great. 

I have two other changes that we might as well tackle while the Readme is being updated. Can we change the link to documentation website from http://cfpb.github.io/eregulations/ to http://cfpb.github.io/eRegulations/ and change the note about the html runner?

Thanks!

**Comment by ascott1** [comment]
:+1: on it

**Comment by ascott1** [comment]
Aaaaaaand done! :boom: Let me know what you think of the README changes in general.

**Comment by KimberlyMunoz** [comment]
I think they look great. It really makes more sense to talk about environment setup up front than to make people dig for it. 

I'll merge as soon as Travis finishes!

**Comment by ascott1** [comment]
Excellent. Thanks!

**Pull request merged**